modules/on_gke/README.md

```diff
@@ -2,63 +2,28 @@
 
 This sub-module deploys Forseti on GKE.  In short, this deploys a server container as a Kubernetes [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) and an orchestrator container as a Kubernetes [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/).  Please see the [Forseti Security GKE Concepts](https://forsetisecurity.org/docs/latest/concepts/forseti-on-gke.html) documentation for more details
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-
-## Requirements
-
-Before this module can be used on a project, you must ensure that the following pre-requisites are fulfilled:
-
-1. Terraform and kubectl are [installed](#software-dependencies) on the machine where Terraform is executed.
-2. The Service Account you execute the module with has the right [permissions](#configure-a-service-account).
-3. The Compute Engine, Kubernetes Engine, and Container Registry APIs are [active](#enable-apis) on the project you will launch the cluster in.
-4. If you are using a Shared VPC, the APIs must also be activated on the Shared VPC host project and your service account needs the proper permissions there.
-5. Forseti has been deployed into a GCP project, preferably the same project in which this Forseti on GKE is intended to be installed.
-
-The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active and the necessary Shared VPC connections.
-
-### Software Dependencies
```

Comment:
I don't sure regarding this one as I understand we need to move to CONTIBUTION.md all related to develop /extend modules, and all needed to run module - should stay in readme

Reply:
Maybe I'm wrong, but only the central README file is specified in the document...